frontend/src/App.js

```diff
@@ -120,26 +120,82 @@ function App() {
     hostSiteVisit, setHostSiteVisit,
     industrySpeaker, setIndustrySpeaker
   }
-  let showAdmins = null;
-  if (loggedUser && loggedUser.a_id) {
-    showAdmins = (
+
+
+  /* ACCESS STRATEGY ( Admins, Staff, Volunteers, Fellows )
+
+  VOLUNTEERS PAGE/DASHBOARD: Admins, Staff
+  => possible alternate YOUR MENTOR(S) PAGE: Fellows
+  
+  EVENTS PAGE/DASHBOARD: All
+  
+  FELLOWS PAGE/DASHBOARD: Admins, Staff
+  => possible alternate YOUR MENTEE(S) PAGE: Volunteers
+  
+  ADMIN TOOLS (edit app users, edit cohorts, edit volunteer skills): Admins
+  */
+
+
+  /* IS VARIABLE: determines user type and assigns simple variable */
+  const appRoute = {};
+  if (loggedUser && loggedUser.admin) {
+    appRoute["admin"] = true;
+  } else if (loggedUser && loggedUser.a_id) {
+    appRoute["staff"] = true;
+  } else if (loggedUser && loggedUser.v_id) {
+    appRoute["volunteer"] = true;
+  } else {
```

Comment:
I will rather using another else if here, I think the way it's set it will default to a fellow user even if there is no loggedUser. I know there are catches for loggedUser further in most pages, if not all but it's better to play it safe

Reply:
ok, added